vllm/config.py

```diff
@@ -1644,6 +1644,11 @@ class ObservabilityConfig:
     """Configuration for observability."""
     otlp_traces_endpoint: Optional[str] = None
 
+    # If set, collects the model forward time for the request. This introduces a
+    # possibly blocking operation to accurately collect the GPU time. It can
+    # have a performance impact on the request latency.
+    collect_model_forward_time: bool = False
```

Comment:
can we raise an exception if it is true and tracing is not enabled?

Reply:
Done.